### ISSUE TITLE
nowIndicator on very wide screens

### ISSUE THREAD

**Issue by sparh** [fullcalendar/fullcalendar-scheduler#130]
Hello,

Set your body margin to 0.
Lload the scheduler width a screen large enough to display it without horyzontal scroll-bar
-> the nowIndicator is not at the right place (something like 30 minutes before now).

Here is a working example http://jsbin.com/quwulowupu/edit?output

- if you wait a few seconds the nowIndicator finally go to the right place
- If you navigate and come back to today the nowIndicator finally go to the right place
- remove the margin:0; in the css and reload. The nowIndicator finally go to the right place

**Comment by arshaw** [comment]
hi @sparh i can't recreate using your exact steps, but i can recreate another way... i just resize the browser window and the now-indicator becomes incorrectly positioned. i'm am confident that if i fix this issue (update on window resize or scrollbar update) it will fix your issue.

**Comment by sparh** [comment]
Hello,
thanks to have checked.

to reproduce it the way I describe it, you must have a screen large enough to display the calendar without horyzontal scroll-bar on page load. (tested on a 22'' and 1920x1080 screen / 15'' and 3840x2160, Chrome, Firefox & IE).
I have tried to resize the browser window after page load and the nowIndicator stay incorrectly positionned.

I'll keep you informed on my bug when you'll provide a fix to what you described.
Regards

![sans titre](https://cloud.githubusercontent.com/assets/6492050/12880386/3086aff8-ce40-11e5-89f3-58d86df157ae.png)

**Comment by arshaw** [comment]
jsbin with latest version, fixed: http://jsbin.com/fuwuhinipo/1/edit?output

**Comment by arshaw** [comment]
just released in [v1.2.1](https://github.com/fullcalendar/fullcalendar-scheduler/releases/tag/v1.2.1)

**Comment by sparh** [comment]
Hello Adam,
thanks for this release but it does not fix the issue. 
I just clicked on the link you have provided http://jsbin.com/fuwuhinipo/1/edit?output and the now indicator is not at the right place onload (it replaces correctly after a while or if I go forward/backward and then come back to today).

On page load
![0](https://cloud.githubusercontent.com/assets/6492050/13139316/58fd8b84-d62c-11e5-84e6-4be04f3e2791.png)

Few seconds/minute later
![1](https://cloud.githubusercontent.com/assets/6492050/13139390/b358303e-d62c-11e5-9207-4596d032e238.png)

**Comment by arshaw** [comment]
if you slightly resize your browser window after you see the problem appear, does it fix the problem?

**Comment by sparh** [comment]
Hello,
yes.
- If I resize the window the NowIndicator replaces itself correctly.
- if I click on Next/Prev button and then I come back to the current day, the NowIndicator replaces itself correctly
- if I wait seconds/minutes the current day, the NowIndicator replaces itself correctly

I think the problem comes from big screen and/or big screen resolution, here is what I have tested and the results (all tested with Chrome, Firefox and IE) : 
15' screen - 1600*900px -> NowIndicator is at the right place (see image below)
15' screen - 3240*2160px -> not good (see image below)
22' screen - 1920x1080px -> not good
27' screen - 1920x1080px -> not good

Check the top right corner
![0](https://cloud.githubusercontent.com/assets/6492050/13212712/a67e974e-d943-11e5-8b84-d20e842a8fee.png)

Check the top right corner
![1](https://cloud.githubusercontent.com/assets/6492050/13212726/c43088e2-d943-11e5-9409-8e5d1dd8a7fa.png)

Hope this helps

**Comment by sparh** [comment]
Hello,
I just want to point another thing : 
on the CSS, if you remove the margin:0; the bug is resolved. http://jsbin.com/yetolokune/edit?output

May it helps!

**Comment by arshaw** [comment]
for me, if the `margin:0` is removed from the body's CSS, the I am actually able to *recreate the bug now* (the opposite of what @sparh said).

this is likely due to very specific screen sizes that we all have, and thus appears random. it likely involved the vertical scrollbars that appear when the resources render, affecting the available width.

i will try to give a fix for this.

**Comment by sparh** [comment]
Hi,
I just re checked and with my laptop (Dell xps 2015 Windows 10, 15' screen - 3240x2160px or 1920x1080px) 
http://jsbin.com/yetolokune/edit?output (no margin -> nowindicator is ok)
http://jsbin.com/fuwuhinipo/1/edit?output (margin:0 -> nowindicator not at the right place)

I noticed that on load nowindicator is at the right place (before ressources are loaded), but once resssources are loaded, the now indicator moves and therefore is not correctly positionned.

On the same laptop, if I set the resolution to 1600x900px, it is ok!

That is very strange!
